middlewares/validators/skills.js

```diff
@@ -0,0 +1,20 @@
+const joi = require("joi");
+
+async function awardSkill(req, res, next) {
+  const schema = joi.object().strict().keys({
+    name: joi.string().required(),
+    by: joi.string().required(),
+    on: joi.string().optional(),
+    for: joi.string().optional(),
+  });
+
+  try {
+    await schema.validateAsync(req.body);
+    next();
+  } catch (error) {
+    logger.error("Error posting skill data: ", error);
```

Comment:
I think in the log, instead of this message "Error posting..", "Error in validating...", would make more sense. What do you think? and in all the other validators in this module, template literal is used in the logger error message, So it would be better that we use template literals in this message also, to maintain consistency